code/datums/elements/embed.dm

```diff
@@ -29,19 +27,16 @@
 	var/jostle_pain_mult
 	var/pain_stam_pct
 	var/payload_type
-	var/embed_chance_turf_mod
 
-/datum/element/embed/Attach(datum/target, embed_chance, fall_chance, pain_chance, pain_mult, remove_pain_mult, impact_pain_mult, rip_time, ignore_throwspeed_threshold, jostle_chance, jostle_pain_mult, pain_stam_pct, embed_chance_turf_mod, projectile_payload=/obj/item/shard)
+/datum/element/embed/Attach(datum/target, embed_chance, fall_chance, pain_chance, pain_mult, remove_pain_mult, impact_pain_mult, rip_time, ignore_throwspeed_threshold, jostle_chance, jostle_pain_mult, pain_stam_pct, projectile_payload=/obj/item/shard)
 	. = ..()
 
 	if(!isitem(target) && !isprojectile(target))
 		return ELEMENT_INCOMPATIBLE
 
 	if(isitem(target))
-		RegisterSignal(target, COMSIG_MOVABLE_IMPACT_ZONE, .proc/checkEmbedMob)
-		RegisterSignal(target, COMSIG_MOVABLE_IMPACT, .proc/checkEmbedOther)
+		RegisterSignal(target, COMSIG_MOVABLE_IMPACT_ZONE, .proc/checkEmbed)
 		RegisterSignal(target, COMSIG_ELEMENT_ATTACH, .proc/severancePackage)
-		RegisterSignal(target, COMSIG_PARENT_EXAMINE, .proc/examined)
```

Comment:
Don't see any changes to examine/impact here ?

Reply:
Oh shoot, I mixed up embed and embedded, one moment